### ISSUE TITLE
import users to AWS Cognito

### ISSUE THREAD

**Issue by hao-fang** [acl-org/acl-2020-virtual-conference#57]
**Number of Volunteers**: 1

- [ ] ~~Get the necessary permission to run import job (ask Nitin)~~ (I prefer to use AWS CLI or boto3 to create the users.)
    * https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pools-creating-import-job.html
- [ ] Get the list of registered users (ask Priscilla)
- [ ] Prepare a CSV file to be uploaded to AWS Cognito (using AWS CLI or boto3)
    * create an user via CLI
   ```bash
    aws cognito-idp admin-create-user \
       --user-pool-id us-east-1_xxxx \
       --username acl2020virtual@gmail.com \
       --user-attributes Name=email,Value=acl2020virtual@gmail.com
    ```
   * boto3: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/cognito-idp.html
- [ ] (Waiting for Priscilla) Need to understand how registration works so we can handle last-minute registration automatically. See issue #55 

Other useful commands
* https://docs.aws.amazon.com/cli/latest/reference/cognito-idp/admin-update-user-attributes.html

**Comment by stupid-2020** [comment]
Dear hao-fang

As I can't find the related closed issue (The RocketChat account"), maybe we can update "user-attributes" of "CLI command" for creating users accordingly as well.

Thank you very much for your help and attention.

**Comment by hao-fang** [comment]
> Dear hao-fang
> 
> As I can't find the related closed issue (The RocketChat account"), maybe we can update "user-attributes" of "CLI command" for creating users accordingly as well.
> 
> Thank you very much for your help and attention.

@stupid-2020 Thanks. I closed that issue #60 because I figured it out with your suggestion :) 
Yes, we can use the CLI command to create users. I'm going to upload a bash script and I'll add you as the reviewer.
But we will need to make a script to process a file rather than adding users one by one.

**Comment by hao-fang** [comment]
🎉
